error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -805,6 +809,20 @@ Flux<T> after(Flux<? extends Iterable<T>> flux, int prefetch) {
     }
   }
 
+  /** Prefer {@link Flux#count()} over collecting into a list and counting its elements. */
+  static final class FluxCount<T> {
+    @BeforeTemplate
+    Mono<Integer> before(Flux<T> flux) {
+      return flux.collect(toImmutableList())
```

Comment:
With [`CollectorMutability`](https://github.com/PicnicSupermarket/error-prone-support/blob/6b4fba62da0e53834c839174ef84da29afd386cb/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/CollectorMutability.java) we also suggest `toCollection(ArrayList::new)` at times; let's add that case.